tests/phpunit/test-wc-stripe-payment-gateway.php

```diff
@@ -631,45 +631,45 @@ public function test_render_subscription_payment_method() {
 	 */
 	public function test_lock_order_payment() {
 		$order_1 = WC_Helper_Order::create_order();
-		$locked  = $this->gateway->lock_order_payment( $order_1 );
+		$locked  = $order_1->lock_payment();
 
 		$this->assertFalse( $locked );
 		$current_lock = $order_1->get_meta( '_stripe_lock_payment' );
 		$this->assertEqualsWithDelta( (int) $current_lock, ( time() + 5 * MINUTE_IN_SECONDS ), 3 );
 
-		$locked = $this->gateway->lock_order_payment( $order_1 );
+		$locked = $order_1->lock_payment();
 		$this->assertTrue( $locked );
 
 		// lock with an intent ID.
 		$order_2   = WC_Helper_Order::create_order();
 		$intent_id = 'pi_123intent';
 
-		$locked       = $this->gateway->lock_order_payment( $order_2, $intent_id );
+		$locked       = $order_2->lock_payment( $intent_id );
 		$current_lock = $order_2->get_meta( '_stripe_lock_payment' );
```

Comment:
Can we update this and any remaining direct meta access to use `get_lock_payment` like you've already done below in line 662

Reply:
Oops, I missed some occurrences of meta retrieval/update. I replaced them in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4086/commits/3e5dffab62656ec385119d409272e13f521c52df . Thanks for flagging!